test/index.test.ts

```diff
@@ -939,4 +939,19 @@ describe('FiatConnect SDK', () => {
       )
     })
   })
+  describe('Cookie Handling', () => {
```

Comment:
To stay consistent with other tests, we should update login tests to check if cookie jar is set and getCookies test just checks the return value from cookie jar. Although I am in process of refactoring login related stuff into a more generic SiweClient on #59, so I can address this on my PR.